packages/page-accounts/src/modals/LocalAdd.tsx

```diff
@@ -0,0 +1,132 @@
+// Copyright 2017-2024 @polkadot/app-accounts authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import type { Blockchain } from '@acala-network/chopsticks-core';
+import type { ActionStatus } from '@polkadot/react-components/Status/types';
+import type { ModalProps } from '../types.js';
+
+import { setStorage } from '@acala-network/chopsticks-core';
+import React, { useCallback, useState } from 'react';
+
+import { Button, Input, InputAddressSimple, Modal } from '@polkadot/react-components';
+import { useApi } from '@polkadot/react-hooks';
+import { keyring } from '@polkadot/ui-keyring';
+
+import { useTranslation } from '../translate.js';
+
+interface Props extends ModalProps {
+  className?: string;
+  onClose: () => void;
+  onStatusChange: (status: ActionStatus) => void;
+}
+
+interface CreateOptions {
+  balance: number;
+  name: string;
+  tags?: string[];
+}
+
+export async function createLocalAccount (chain: Blockchain, address: string, { balance, name }: CreateOptions): Promise<ActionStatus> {
+  const status = { action: 'create' } as ActionStatus;
+
+  try {
+    keyring.addExternal(address, { isLocal: true, name, tags: ['local'] });
+
+    await setStorage(chain, {
+      System: {
+        Account: [
+          [[address], { data: { free: balance * 1e12 }, providers: 1 }]
```

Comment:
it will be wise to not do this for mocked account because this will reset the state. Account has additional data that might be important. I suggest we don't modify it. If funds needed they can come from Alice account which has plenty of funds or the dev can call `setStorage` PRC and modify this.

Reply:
I see, will have PR this remove this